### PR TITLE
Add Password Form Component

### PR DESCRIPTION
**Pull request by ykaya932**
- Created new component for password
- New component checks whether the password is strong or not. If it is weak password, it shows the requirements of the strong password
Screenshot from 2020-11-24 00-21-34

### CODE REVIEW

File: web/carousel/src/components/SignUp/CustomerSignup.js [hunk 1]
```diff
@@ -17,6 +19,7 @@ class CustomerSignUp extends Component {
       surname: "",
       errMessage: "",
       signUpMessage: "",
+      wasWeak: true
```
Comment (afraarslan) [inline web/carousel/src/components/SignUp/CustomerSignup.js]: I think it should be `isWeakPassword` as in VendorSignup.js? 🤔

File: web/carousel/src/components/SignUp/CustomerSignup.js [hunk 2]
```diff
@@ -32,13 +35,15 @@ class CustomerSignUp extends Component {
...
-              floatingLabelText="Password"
+            <PasswordForm
+              password={this.state.password}
               onChange={(event, newValue) =>
                 this.setState({ password: newValue })
               }
+              setIsWeak={(newValue) => {
+                this.setState({isWeakPassword: newValue});
+              }}
+              wasWeak={this.state.isWeakPassword}
```
Comment (afraarslan) [inline web/carousel/src/components/SignUp/CustomerSignup.js:46]: Here we call `isWeakPassword` but at the bottom, it is `this.state.wasWeak`

File: web/carousel/src/components/SignUp/CustomerSignup.js [hunk 3]
```diff
@@ -71,6 +76,9 @@ class CustomerSignUp extends Component {
   }
   handleCustomerSignUp(event) {
     var self = this;
+    if (this.state.wasWeak) {
```
Comment (afraarslan) [inline web/carousel/src/components/SignUp/CustomerSignup.js]: `this.state.isWeakPassword`

File: web/carousel/src/components/PasswordForm/PasswordForm.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    
+
+    render() {
+
+        const errorMessage = this.checkPassword(this.props.password);
+
+        let errorMessageLine = null;
+
+        console.log("HEYYY");
```
Comment (afraarslan) [inline web/carousel/src/components/PasswordForm/PasswordForm.js]: Heyy! ✋

File: web/carousel/src/components/SignUp/CustomerSignup.js (hunk 1 above)
Comment (oncelkeles) [reply web/carousel/src/components/SignUp/CustomerSignup.js]: I think so too

Comment (ykaya932) [reply web/carousel/src/components/SignUp/CustomerSignup.js]: Yes, my bad. You are right :+1:

**Review by ahmetyigitgedik — APPROVED** [review]
Nice job 💯

**Pull request merged**